### PR TITLE
Show in the console two or more weapons skins using the config #141

### PR DESCRIPTION
**Pull request by JanEickholt**
just #161, but in a new branch now

### CODE REVIEW

File: src/constants.py [hunk 1]
```diff
@@ -154,8 +154,24 @@
...
+                "VALORANT GO! Vol. 1": "V. G. Vol. 1",
+                "VALORANT GO! Vol. 2": "V. G. Vol. 2",
+                "Radiant Crisis 001": "RC 001",
+                "Sentinels of Light": "SoL",
+                "Prelude to Chaos": "PtC",
+                "Winterwunderland": "Winter",
+                "Kohaku & Matsuba": "K&M",
+                "Gaia's Vengeance": "Gaia"
+            }
+        },
```
Comment (zayKenyon) [inline src/constants.py:174]: Previously discussed: [here](https://github.com/zayKenyon/VALORANT-rank-yoinker/pull/161#issuecomment-1658976073).

File: src/config.py [hunk 2]
```diff
@@ -53,10 +53,12 @@ def __init__(self, log):
 
             self.log(f"got cooldown with value '{self.cooldown}'")
 
-            if not self.weapon_check(config["weapon"]):
-                self.weapon = "vandal" # if the user manually entered a wrong name into the config file, this will be the default until changed by the user.
-            else:   
-                self.weapon = config["weapon"]
+            weapons = [weapon.strip() for weapon in config["weapons"].split(",")]
+            if not self.weapon_checks(weapons):
+                self.weapons = ["vandal"] # if the user manually entered a wrong name into the config file, this will be the default until changed by the user.
```
Comment (zayKenyon) [inline src/config.py:58]: Could we change this to by Vandal and Phantom by default?

**Review by zayKenyon — CHANGES_REQUESTED** [review]
As part of the config onboarding (when you run the app with the `--config` flag), users can input a weapon via InquirerPy, could you change this so they can make 2 inputs?

File: src/constants.py (hunk 1 above)
Comment (ThisIsTeddyBear) [reply src/constants.py:174]: You can also add Tethered Realms, Nunca Olvidados, Radiant Crisis 001, Protocol 781-A, Divine Swine, Lycan's Bane, Coalition Cobra, Task Force 809, Piedra del Sol, Premiere Collision, Starlit Odyssey.
There are many long skin names making all of them short will make it confusing but we need to have a good system for this other wise there wont be space left in the table.

**Comment by JanEickholt** [comment]
> As part of the config onboarding (when you run the app with the `--config` flag), users can input a weapon via InquirerPy, could you change this so they can make 2 inputs?

I am unsure about the elegance of the code, but you can add multiple weapons now

File: src/constants.py (hunk 1 above)
Comment (JanEickholt) [reply src/constants.py]: > You can also add Tethered Realms, Nunca Olvidados, Radiant Crisis 001, Protocol 781-A, Divine Swine, Lycan's Bane, Coalition Cobra, Task Force 809, Piedra del Sol, Premiere Collision, Starlit Odyssey.

I added all of them, that are longer than 10 characters

> There are many long skin names making all of them short will make it confusing

I understand your point, but I would still appreciate suggestions on making it possible

**Pull request closed without merging**